src/main/java/uk/gov/hmcts/reform/civil/service/judgments/CjesService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.civil.service.judgments;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.client.CjesApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.judgmentonline.cjes.JudgmentDetailsCJES;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+
+public class CjesService {
+
+    private final CjesApiClient cjesApiClient;
```

Comment:
I can see this error:
![image](https://github.com/user-attachments/assets/3176a99a-c431-429d-83d5-46d9ec83ced7)